lib/shared/translation/locale_keys.dart

```diff
@@ -26,4 +31,13 @@ class LocaleKeys{
   static const noUserFoundEmail = 'no_user_found_email';
   static const wrongPasswordProvided = 'wrong_password_provided';
   static const facebookLoginFailed = 'facebook_login_failed';
+}
+
+class Routes{
```

Comment:
Why this class in the locale_keys file ?

Reply:
now separated into routes.dart